plugins/rts54hub/fu-rts54hub-device.c

```diff
@@ -25,7 +25,7 @@ G_DEFINE_TYPE(FuRts54HubDevice, fu_rts54hub_device, FU_TYPE_USB_DEVICE)
 #define FU_RTS54HUB_DEVICE_TIMEOUT_ERASE 5000  /* ms */
 #define FU_RTS54HUB_DEVICE_TIMEOUT_AUTH	 10000 /* ms */
 #define FU_RTS54HUB_DEVICE_BLOCK_SIZE	 4096
-#define FU_RTS54HUB_DEVICE_STATUS_LEN	 25
+#define FU_RTS54HUB_DEVICE_STATUS_LEN	 24
```

Comment:
so this changes the size of the buffer -- this won't regress on older hardware or firmware versions?

Reply:
The status of Gen2 hub, index [24] this BYTE we don't used on the update flow.
In order to this update flow on FWUPD can use on Gen1 and Gen2 hub of realtek  so changed STATUS-LEN to 24.
This change not affect older hardware or firmware versions.